web/common/Header.js

```diff
@@ -245,7 +245,7 @@ export function NavigationMenu({ open, setOpen }) {
           )
           .map(route => (
             <ListRouteItem
-              key={route.path || route.href}
+              key={route.path || route.label}
```

Comment:
Are we sure `route.label` is unique?

Reply:
No I'm not sure how to solve this issue ?
But I think I understood that `route.href` is undefined, so it was clearly not a good candidate for a key...